c_emulator/riscv_platform.c

```diff
@@ -21,6 +21,17 @@ bool sys_enable_fdext(unit u)
 bool sys_enable_zfinx(unit u)
 { return rv_enable_zfinx; }
 
+/* When Smepmp spec is ratified, remove the 'experimental' conditionals */
+bool sys_enable_smepmp(unit u)
+{ return ( (rv_enable_smepmp == 1) && rv_enable_experimental_extensions ); }
```

Comment:
rv_enable_smepmp and rv_enable_zicond are now Boolean values, so we can just do the and without the comparison to 1 and all the parenthesis

Reply:
Fixed and pushed onto the branch.